packages/web3-errors/src/web3_error_base.ts

```diff
@@ -17,9 +17,9 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 
 /* eslint-disable max-classes-per-file */
 
-import { Web3Error as ErrorInterface } from 'web3-types';
+import { Web3ErrorInterface } from 'web3-types';
 
-export abstract class Web3Error extends Error implements ErrorInterface {
+export abstract class Web3Error extends Error implements Web3ErrorInterface {
```

Comment:
small nit: maybe it would make more sense to name this BaseWeb3Error and leave Web3Error to be interface as users would you this as a type in catch clauses. No strong preference though